docs/network-flow-visibility.md

```diff
@@ -37,55 +48,61 @@ Services, Pods etc., and thereby provides opportunities to enhance the performan
 and security aspects of Pod workloads.
 
 For visualizing the network flows, Antrea monitors the flows in Linux conntrack
-module. These flows are converted to flow records and are sent to the configured
-flow controller. High-level design is given below:
+module. These flows are converted to flow records, and then flow records are post-processed
+before they are sent to the configured external flow collector. High-level design is given below:
 
-![Flow Exporter Design](assets/flow_exporter.svg)
+![Antrea Flow Visibility Design](assets/flow_visibility.svg)
```

Comment:
what does the arrow direction between flow exporter and conntrack mean? I would suppose flow exporter to get data from conntrack

Reply:
Yes, it signifies that flow exporter is polling conntrack module to get flow data.